fort_setup.sh

```diff
@@ -78,11 +78,20 @@ if [ "$?" = "0" ] ; then
   TMP=""
 fi
 
+# Get the configuration directory
+CONF_DIR="$(dirname "${TALS_LOC}")"
+if [ ! -w "${CONF_DIR}" ]; then
+  CONF_DIR="/tmp/fort/"
```

Comment:
This change doesn't affect the functionality, is merely a detail. Please remove the last '/', should result in something like this:
```
CONF_DIR="/tmp/fort"
```

Reply:
No problem, I changed it and rebased the commit

Reply:
Thanks @alarig! That'll do.